hydra-node/hydra-node.cabal

```diff
@@ -178,6 +179,7 @@ library
     , typed-protocols
     , typed-protocols-cborg
     , typed-protocols-examples
+    , unliftio
```

Comment:
Used `writeBinaryFileDurableAtomic` from there to ensure files are written (it writes to a temp file and moves it after write, which is atomic).

Reply:
given this is a "Dirt road" feature, does it really matter? Couldn't we just use plain old BS `writeFile` functions? This would remove the need to add a dependency.

Reply:
Also, having read articles in the past on how hard it is to actually ensure something is written on disk(https://lwn.net/Articles/457667/) and the great length to which SQLite goes to provide atomicity (https://www.sqlite.org/atomiccommit.html) I am doubtful unliftio provides the kind of guarantees we really would like it to provide so I would suggest to use simpler stuff and then later refactor to use SQLite.